File: C# Part 2/03.Methods/03.EnglishDigit.cs
using System;
using System.Linq;

namespace EnglishDigit
{
    class EnglishDigit
    {
        static void Main()
        {
            DigitToWord( Console.ReadLine().ToCharArray().LastOrDefault() - '0');
        }

        public static void DigitToWord(int digit)
        {
            string[] numbers = { "zero", "one", "two", "three", "four", "five", "six", "seven", "eight", "nine" };
            Console.WriteLine(numbers[digit]);
        }
    }
}

File: JavaScript Fundamentals/08. Functions/03_EnglishDigit.js
function solve(args) {
    var input = args.toString().split("");
    var numToWord = ['zero', 'one', 'two', 'three', 'four', 'five', 'six', 'seven', 'eight', 'nine'];

    console.log(numToWord[+input[input.length - 1]]);
}


solve(42);